lib/services/debug-data-service.ts

```diff
@@ -0,0 +1,43 @@
+export class DebugDataService implements IDebugDataService {
+	constructor(private $projectData: IProjectData,
+		private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants,
+		private $platformService: IPlatformService,
+		private $platformsData: IPlatformsData) { }
+
+	public createDebugData(debugService: IPlatformDebugService, options: IOptions, buildConfig: IBuildConfig): IDebugData {
+		this.$projectData.initializeProjectData(options.path);
+		return {
+			applicationIdentifier: this.$projectData.projectId,
+			projectDir: this.$projectData.projectDir,
+			deviceIdentifier: options.device,
+			pathToAppPackage: this.getPathToAppPackage(debugService, options, buildConfig),
+			projectName: this.$projectData.projectName
+		};
+	}
+
+	private getPathToAppPackage(debugService: IPlatformDebugService, options: IOptions, buildConfig: IBuildConfig): string {
+		if (debugService.platform === this.$devicePlatformsConstants.Android) {
```

Comment:
I think it'd be better to use `$mobileHelper.isAndroidPlatform` here in case the casing differs